### ISSUE TITLE
'Checkout to' List

### ISSUE THREAD

**Issue by Canceltodebug** [snipe/snipe-it#1786]
Issue regarding the 'Checkout to' drop list when creating / assigning an asset.

#### Expected Behavior

The 'Checkout to' list of people to be in alphabetical order.

#### Actual Behavior

The list is showing all users that were imported into the system first and then the manually added people afterwards. Image attached.

![image](https://cloud.githubusercontent.com/assets/16987693/13180213/6eacdca2-d71f-11e5-9525-b7148e730300.png)

#### Before posting this issue, confirm/answer the following:
- [ ] I have enabled debug mode
- [x] I have read [checked the Common Issues page](http://docs.snipeitapp.com/common-issues.html)
- [x] I have read [the Getting Help Documentation](http://docs.snipeitapp.com/getting-help.html)

**Comment by snipe** [comment]
I'm not actually sure how that's possible. The `getCheckout` method uses a helper function called `usersList()`, which does the following:

```
  public static function usersList() {
      $users_list = array('' => trans('general.select_user')) + DB::table('users')
      ->select(DB::raw('concat(last_name,", ",first_name," (",username,")") as full_name, id'))
      ->whereNull('deleted_at')
      ->orderBy('last_name', 'asc')
      ->orderBy('first_name', 'asc')
      ->pluck('full_name', 'id');
      return $users_list;
  }
```

Is it possible that your imported users had a space in front of their last names? That would cause the orderBy to put them first.

**Comment by Canceltodebug** [comment]
Sorry for the late reply.
I have checked the file I used for the original bulk upload and there is a space in the CSV file between the first and last name for each record. I believe that I followed the correct procedure when creating the file.
I've accessed the database and can see that for the uploaded records, the lastname field is prefixed by a space. Manually added users do not have this so I assume that this is the reason for this issue.
The odd thing is that Snipe-IT does not show this 'extra' space in the people properties.

I'm not going to worry about it as I will be exporting all data at some point and upload into Snipe v3 when I get around to installing it and can make any amendments at that time.
